visitor/src/test/java/com/iluwatar/visitor/SoldierTest.java

```diff
@@ -42,7 +42,8 @@ public SoldierTest() {
 
   @Override
   void verifyVisit(Soldier unit, UnitVisitor mockedVisitor) {
-    verify(mockedVisitor).visitSoldier(eq(unit));
+    //CS304 (manually written) Issue link: https://github.com/iluwatar/java-design-patterns/issues/1968
```

Comment:
Remove this comment

Reply:
Thanks a lot for mention that, I have removed these comment, sorry about it.